pkg/kubelet/util/util_unsupported.go

```diff
@@ -52,3 +52,8 @@ func LocalEndpoint(path, file string) (string, error) {
 func GetBootTime() (time.Time, error) {
 	return time.Time{}, fmt.Errorf("GetBootTime is unsupported in this build")
 }
+
+// Umask empty implementation
+func Umask(mask int) (int, error) {
+	return 0, errors.New("platform and architecture is not supported")
```

Comment:
Can you share more about the usage of `errors.New` instead of `fmt.Errorf`? Looks like the other entries in the file almost entirely use `fmt.Errorf`.

Reply:
Changes made